modules/pxp-module-puppet

```diff
@@ -33,8 +33,8 @@ env_fix_up = nil
 self.class.send(:define_method, :get_env_fix_up) do
   # Prepare an environment fix-up to make up for its cleansing performed
   # by the Puppet::Util::Execution.execute function.
-  # Without this fix-up it's impossible to run puppet under a non-root
-  # user because it cannot find the user's HOME directory.
+  # This fix-up is meant for running puppet under a non-root user;
+  # pxp-module-puppet cannot find the user's HOME directory otherwise.
```

Comment:
it's thge puppet agent who crashes if it can't find the user's HOME directory, not the pxp-module-puppet